src/cookie/cookie.ts

```diff
@@ -13,6 +13,9 @@ export class Cookie {
   public rejectCookies = document.getElementsByName("cookiesReject");
   cookieDomain: string;
 
+  HIDDING_CLASS = "govuk-!-display-none";
```

Comment:
Makes sense - I think this may be a typo `HIDING_CLASS` 
But i think for the naming to be improved a little further, the naming could be  either `HIDDEN_CLASS` and `SHOWN_CLASS` or `HiddenElement` or `ShownElement` 

Let me know what you think?